connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/PlanGenerationTestSuite.scala

```diff
@@ -3289,7 +3289,7 @@ class PlanGenerationTestSuite
             .setUnparsedIdentifier("id")))
       .setCustomField("abc")
       .build()
-    simple.select(column(_.setExtension(com.google.protobuf.Any.pack(extension))))
+    simple.select(Column(_.setExtension(com.google.protobuf.Any.pack(extension))))
```

Comment:
Why don't we just have `column` method?

Reply:
It is more to illustrate that the old way works. You can actually use both.